### PR TITLE
233 admin statistics

### PR DESCRIPTION
**Pull request by pferdefleisch**
Comments in the commits https://github.com/where2help/where2help/commit/e880a303a19bd742fdf88782b692c8c6e225f889

### CODE REVIEW

File: app/admin/concepts/statistic/operation.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+class StatisticOperation
+  class EventCount
+    class Show
+      def call(params = {})
+        Event.count
+      end
+    end
+  end
+
+  class TotalParticipants
```
Comment (klappradla) [inline app/admin/concepts/statistic/operation.rb]: Just to make sure: are you on purpose not distinguishing between `pending` and `published` Events here?

Don't know to which extend this *feature* is actually used though.

File: app/admin/resources/dashboard.rb [hunk 2]
```diff
@@ -21,7 +21,36 @@
     #       end
     #     end
     #   end
+    columns do
```
Comment (klappradla) [inline app/admin/resources/dashboard.rb]: Love it 💚 
Need help for the translations?

Comment (pferdefleisch) [inline app/admin/resources/dashboard.rb]: Yes please.

File: app/admin/concepts/statistic/operation.rb (hunk 1 above)
Comment (pferdefleisch) [inline app/admin/concepts/statistic/operation.rb]: I'll update this tomorrow. If I have time. Tomorrow is the Chatbot Summit that I stayed here so long for.

Comment (klappradla) [inline app/admin/concepts/statistic/operation.rb]: Oh, good point about the `volunteers_count` @pferdefleisch, did not think about this. Totally fine then to not be more explicit about I'd say. But as you like, maybe it's a useful reminder in some months then.

File: app/admin/resources/dashboard.rb (hunk 2 above)
Comment (pferdefleisch) [inline app/admin/resources/dashboard.rb]: I think we can merge it. But we should also create an issue to add the translations.
You can either do both or delegate to me for the latter.
bussi :kiss:

Comment (klappradla) [inline app/admin/resources/dashboard.rb]: Gonna add them tomorrow at Ruby Habits and merge then.

**Comment by klappradla** [comment]
From looking at this yesterday, tiny question just out of curiosity @pferdefleisch: 

Why do you always use `Class`, never `Module`? Is this the Trailblazers style? E.g.
```ruby
class StatisticOperation
  class EventCount
    class Show
      def call(params = {})
        Event.count
      end
    end
  end
...
```

**Comment by pferdefleisch** [comment]
I only use modules when I'm either sure it's just a namespace or then I want to use it as a mixin. Neither are true here. Personal preference

**Pull request merged**